api/src/main/resources/Appointment.hbm.xml

```diff
@@ -62,6 +62,13 @@
             <one-to-many class="AppointmentAudit"/>
         </set>
 
+        <set name="encounters" table="patient_appointment_encounter_map" lazy="true" cascade="none">
+            <key column="patient_appointment_id" />
+            <many-to-many class="org.openmrs.Encounter">
+                <column name="encounter_id" />
+            </many-to-many>
+        </set>
```

Comment:
I think we need to be really careful here, and ensure our tests cover the various cascading cases that exist, both from the Hibernate perspective as well as the OpenMRS saveHandlers perspective.  i.e. we don't necessarily want to to find that voiding an Appointment ends up voiding all of the associated Encounters.  Or that deleting it from the database deletes the Encounter rows as well.

Reply:
Fair enough, I did set cascade=none, but good to actually test what this does, I will add some tests.